From 1c2b82bcd6cd3d8e46e6360b0f43079c8fb3899f Mon Sep 17 00:00:00 2001
From: mbrinkma <mbrinkma@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Wed, 3 Nov 2021 17:16:15 +0000
Subject: [PATCH 434/462] Mantis 0061820: tests shall be able to run with kiosk
 "qa-prod-automate" - pass "-DsbdTests.profile=..." to tests (which are run in
 a separate VM which leads to loss of all system-properties)

git-svn-id: svn://localhost/SelfServiceBuild/trunk@820 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/scripts/java8/gradlew | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/SelfServiceBuild/Build/scripts/java8/gradlew b/SelfServiceBuild/Build/scripts/java8/gradlew
index 4b64ca2..86a0167 100755
--- a/SelfServiceBuild/Build/scripts/java8/gradlew
+++ b/SelfServiceBuild/Build/scripts/java8/gradlew
@@ -161,10 +161,10 @@ save () {
     for i do printf %s\\n "$i" | sed "s/'/'\\\\''/g;1s/^/'/;\$s/\$/' \\\\/" ; done
     echo " "
 }
-APP_ARGS=$(save "$@")
+export CMD_LINE_ARGS=$(save "$@")
 
 # Collect all arguments for the java command, following the shell quoting and substitution rules
-eval set -- $DEFAULT_JVM_OPTS $JAVA_OPTS $GRADLE_OPTS "\"-Dorg.gradle.appname=$APP_BASE_NAME\"" -classpath "\"$CLASSPATH\"" org.gradle.wrapper.GradleWrapperMain "$APP_ARGS"
+eval set -- $DEFAULT_JVM_OPTS $JAVA_OPTS $GRADLE_OPTS "\"-Dorg.gradle.appname=$APP_BASE_NAME\"" -classpath "\"$CLASSPATH\"" org.gradle.wrapper.GradleWrapperMain "$CMD_LINE_ARGS"
 
 # by default we should be in the correct project dir, but when run from Finder on Mac, the cwd is wrong
 if [ "$(uname)" = "Darwin" ] && [ "$HOME" = "$PWD" ]; then
-- 
2.41.0.windows.1

